src/stores/game.js

```diff
@@ -334,7 +339,7 @@ export const useGameStore = defineStore('game', {
               this.myPNum = res.pNum;
               this.updateGame(res.game);
               this.successfullyJoined({
-                username: res.playerUsername,
+                username: res.username,
```

Comment:
You could destructure these off red and use the directly to clean up this function some. It’s also allow shorthand when putting keys in this object.
```
{ username, pNum }
```